doc/users/whats_new/2015-07-30_unpack_labeled_data.rst

```diff
@@ -0,0 +1,30 @@
+Working with labeled data like pandas DataFrames
+------------------------------------------------
+Plot methods which take arrays as inputs can now also work with labeled data
+and unpack such data.
+
+This means that the following two examples produce the same plot::
+
+Example ::
+    df = pandas.DataFrame({"var1":[1,2,3,4,5,6], "var2":[1,2,3,4,5,6]})
+    plt.plot(df["var1"], df["var2"])
+
+
+Example ::
+    plt.plot("var1", "var2", data=df)
+
+This works for most plotting methods, which expect arrays/sequences as
+inputs and ``data`` can be anything which supports ``__get_item__``
+(``dict``, ``pandas.DataFrame``,...).
+
+In addition to this, some other changes were made, which makes working with
+``pandas.DataFrames`` easier:
```

Comment:
Should be `DataFrame`, not `DataFrames` if you're putting it in fixed-width font